procstats/proc.go

```diff
@@ -182,6 +191,13 @@ func CollectProcInfo(pid int) (ProcInfo, error) {
 type CPUInfo struct {
 	User time.Duration // user cpu time used by the process
 	Sys  time.Duration // system cpu time used by the process
+
+	// Linux-specific details about the CPU configuration of the process.
```

Comment:
Maybe we could add a reference to https://www.kernel.org/doc/Documentation/scheduler/sched-bwc.txt to provide the reader more doc ?